site.properties

```diff
@@ -89,29 +89,35 @@ ldap.lastNameAttribute  = sn
 
 # Database and JPA (H2)
 
-#jdbc.driver   = org.h2.Driver
-#jdbc.url      = jdbc:h2:../Manager-Repository;DB_CLOSE_DELAY=-1
-#jdbc.username = apromore
-#jdbc.password = MAcri
-#
-#jpa.database         = H2
-#jpa.databasePlatform = org.eclipse.persistence.platform.database.H2Platform
-#jpa.showSql          = false
-#jpa.generateDDL      = false
+jdbc.driver   = org.h2.Driver
+jdbc.url      = jdbc:h2:../Manager-Repository;DB_CLOSE_DELAY=-1;DATABASE_TO_UPPER=false;
+jdbc.username = apromore
+jdbc.password = MAcri
 
+jpa.database         = H2
+jpa.databasePlatform = org.eclipse.persistence.platform.database.H2Platform
+jpa.showSql          = false
+jpa.generateDDL      = false
 
+# Liquibase User H2
+liquibase.jdbc.username = apromore
+liquibase.jdbc.password = MAcri
 
 # Database and JPA (MySQL)
 
-jdbc.driver   = com.mysql.jdbc.Driver
-jdbc.url      = jdbc:mysql://localhost:3306/apromore?createDatabaseIfNotExist=true&autoReconnect=true&allowMultiQueries=true&rewriteBatchedStatements=true&characterEncoding=utf-8
-jdbc.username = apromore
-jdbc.password = MAcri
+#jdbc.driver   = com.mysql.jdbc.Driver
+#jdbc.url      = jdbc:mysql://localhost:3306/apromore?createDatabaseIfNotExist=true&autoReconnect=true&allowMultiQueries=true&rewriteBatchedStatements=true&characterEncoding=utf-8
+#jdbc.username = root
+#jdbc.password = MAcri
+#
+#jpa.database         = MYSQL
+#jpa.databasePlatform = org.eclipse.persistence.platform.database.MySQLPlatform
+#jpa.showSql          = false
+#jpa.generateDDL      = false
```

Comment:
Why don't we use the same 'apromore' user like H2 instead of 'root'?

Reply:
Can it is named "liquibase_user" instead of "db_user" (it is full privileges)? The name will be shown in the database management tool for information as well?

Reply:
I am not sure how it became root. Changeing that back to apromore. It was a mistake.

db_user is just a name. And looking at the Docs of mysql about the privileges, All privileges is the right 1. (i.e ALL privileges does not include create user.)

Reply:
Corrected to liquibase_usr